crates/re_build_tools/src/lib.rs

```diff
@@ -194,7 +184,7 @@ pub fn export_build_info_vars_for_crate(crate_name: &str) {
         // We need to check `IS_IN_RERUN_WORKSPACE` in the build-script (here),
         // because otherwise it won't show up when compiling through maturin.
         // We must also make an exception for when we build actual wheels (on CI) for release.
-        if is_on_ci() {
+        if environment == Environment::RerunCI {
             // e.g. building wheels on CI.
             set_env("RE_BUILD_IS_IN_RERUN_WORKSPACE", "no");
```

Comment:
We should probably do this also on `Environment::PublishingCrates` (which likely is on CI)